src/control/cmd/dmg/pretty/pool.go

```diff
@@ -190,15 +214,28 @@ func printListPoolsResp(out io.Writer, resp *control.ListPoolsResp) error {
 		fmt.Fprintln(out, "no pools in system")
 		return nil
 	}
+	var upgrade bool = false
+	for _, pool := range resp.Pools {
+		if pool.HasErrors() {
+			continue
+		}
+		if pool.PoolLayoutVer != pool.UpgradeLayoutVer {
+			upgrade = true
+		}
+	}
 
-	formatter := txtfmt.NewTableFormatter("Pool", "Size", "State", "Used", "Imbalance", "Disabled")
+	titles := []string{"Pool", "Size", "State", "Used", "Imbalance", "Disabled"}
+	if upgrade == true {
```

Comment:
(style) No need to explicitly compare booleans.
```suggestion
	if upgrade {
```